ls1intum/Artemis
# Added connectors user update to methods

### Checklist

### Motivation and Context


This corresponds to my local fix to the [issue nr. 1847](https://github.com/ls1intum/Artemis/issues/1847). It solves the problem of the incorrect synchronization between Artemis and GitLab of the user's groups, if these are added or removed directly from the course's access groups (see screenshots).

### Description

The fix consists in using the already implemented "updateUserInConnectorsAndAuthProvider(User user, Set<String> oldGroups)" method to update the user's groups in both, Artemis and in the GitLab instance. 

### Steps for Testing


1. Go to 'Course management view'
2. Click on an Access Group of a course
3. Add or remove a user to/from the list
4. Check on the GitLab instance if user has been added/removed to/from the members group of the course's exercises

### Screenshots


![Screenshot from 2020-07-09 21-43-55](https://user-images.githubusercontent.com/45761921/87180592-1f7df500-c2e1-11ea-9bc5-acdbe4297f04.png)
![Screenshot from 2020-07-10 19-12-21](https://user-images.githubusercontent.com/45761921/87180954-c8c4eb00-c2e1-11ea-8a4b-8b471c889230.png)
